njsscan/rules/semantic_grep/database/nosql_find_injection.yaml

```diff
@@ -2,6 +2,8 @@ rules:
 - id: node_nosqli_injection
   patterns:
   - pattern-not-inside: |
+      import $SANITIZE from 'mongo-sanitize'
```

Comment:
You need to move this to a separate `pattern-not-inside:`

```
  - pattern-not-inside: |
        import $SANITIZE from 'mongo-sanitize'
        ... 
        $SANITIZE(...)
         ...
  ```

Reply:
in the same file?

Reply:
Yes under `patterns`

Reply:
Can you check again? Greetings